github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1092,6 +1092,125 @@ periodics:
     - name: pgp-bot-key
       secret:
         secretName: pgp-bot-key
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: prow-workloads
```

Comment:
No need to run on the workloads cluster, given that this job is not actually going to spin up a cluster it is ok to run on `ibm-prow-jobs`

Reply:
Done!